index.js

```diff
@@ -522,11 +522,17 @@ function GenericPLCAccessory(platform, config, accessoryNumber) {
     this.service = new Service.OccupancySensor(this.name);
     this.accessory.addService(this.service);
 
+    this.modBitGet = this.plain

```

Comment:
To be in line with the naming of other accessories and adding the missing ;
```suggestion
    this.modFunctionGet= this.plain;

```